packages/combo-box/test/filtering.test.js

```diff
@@ -372,7 +372,7 @@ describe('filtering items', () => {
 
     it('should properly display all items in the selector', () => {
       comboBox.open();
-      comboBox.filteredItems = Array.apply(null, Array(10)).map((_, i) => `item${i}`);
+      comboBox.filteredItems = Array.from({ length: 10 }, (_, i) => `item${i}`);
```

Comment:
Let's move the new logic to `makeItems` helper in `helpers.js` and then use it for consistency.

Reply:
Done.